l10n_it_fiscalcode/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
```

Comment:
Add this `:target: http://www.gnu.org/licenses/agpl-3.0-standalone.html`, on line 2